src/components/HelpPanel/HelpPanelTabs/SupportPanel.stories.tsx

```
@@ -120,16 +120,16 @@ export const WithCases: Story = {
+      canvas.getByRole('heading', { name: /My open support cases \(21\)/ })
     ).toBeInTheDocument();
     expect(
       canvas.getByText('Insights subscription activation issue')
     ).toBeInTheDocument();
     expect(
       canvas.getByText('API rate limit clarification')
     ).toBeInTheDocument();
-    expect(canvas.getByText('Waiting on Red Hat')).toBeInTheDocument();
-    expect(canvas.getByText('Waiting on Customer')).toBeInTheDocument();
+    expect(canvas.getAllByText('Waiting on Red Hat').length).toBeGreaterThan(0);
+    expect(canvas.getAllByText('Waiting on Customer').length).toBeGreaterThan(0);
```

ESLint

[error] Replace `0` with `⏎······0⏎····`

(prettier/prettier)